### ISSUE TITLE
[inferrer] Add support for "ltree" PG type

### ISSUE THREAD

**Issue by dylanwolff** [rom-rb/rom-sql#145]
We're using the `ltree` PG type in a `pages` table like so:

```
CREATE TABLE pages (
    id integer NOT NULL,
    title text NOT NULL,
    body json DEFAULT '[]'::json NOT NULL,
    path text NOT NULL,
    status text NOT NULL,
    created_at timestamp without time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
    updated_at timestamp without time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
    ltree_path ltree,
    "position" integer DEFAULT 0
);
```
When using an inferred schema for this table, the following error is raised:
```
ROM::Schema::MissingAttributesError: missing attributes in ROM::Relation::Name(pages) schema: :ltree_path
```
We've been able to get around this by augmenting the inferred schema like so:
```
schema :pages, infer: true do
  attribute :ltree_path, Types::Strict::String
end
```
but it'd be great to have built-in support for inferring the "ltree" type.

Thanks!

**Comment by flash-gordon** [comment]
OK, let's add it to the queue :)
https://www.postgresql.org/docs/9.6/static/ltree.html

**Comment by GustavoCaso** [comment]
I can work on this one

**Comment by flash-gordon** [comment]
this is 💯 free and waits for you @GustavoCaso :)

**Comment by solnic** [comment]
This is now available in rom-sql 2.1.0
